tests/parallel/tblocking_channel.nim

```diff
@@ -0,0 +1,34 @@
+import threadpool, os
```

Comment:
No ``discard`` section means this test is not run, it's only checked it compiles.